pkg/cli/init-templates/predict.py

```diff
@@ -0,0 +1,18 @@
+# Prediction interface for Cog ⚙️
+# Reference: https://github.com/replicate/cog/blob/main/docs/python.md
+	
+import cog
+# import torch
+
+class Predictor(cog.Predictor):
+    def setup(self):
+      """Load the model into memory to make running multiple predictions efficient"""
+      # self.model = torch.load("./weights.pth")
+
+    @cog.input("image", type=cog.Path, help="Grayscale input image")
```

Comment:
Maybe we could have a comment pointing to the documentation for the predict() function here, to make it super easy to read about input types, etc.?

Reply:
There's a comment at the top of the file:

```yaml
# Prediction interface for Cog ⚙️
# Reference: https://github.com/replicate/cog/blob/main/docs/python.md
```